src/lib/cms-content.ts

```diff
@@ -0,0 +1,80 @@
+import axios from 'axios';
+
+export type ContentType = 'Case Study' | 'Course' | 'Lab' | 'PDF';
+
+export interface Lesson {
+  title: string;
+  link: string;
+}
+
+export interface ContentItem {
+  title: string;
+  contentType: ContentType;
+  longDescription?: string;
+  slug: string;
+  externalLink?: string;
+  level?: string;
+  durationHours?: number;
+  shortDescription: string;
+  lessons?: Lesson[];
+  fileDownload?: string;
+}
+
+const itemMap = ({
+  title,
+  contentType,
+  longDescription,
+  slug,
+  externalLink,
+  level,
+  durationHours,
+  shortDescription,
+  lessons,
+  fileDownload,
+}: any): ContentItem => {
+  return {
+    title,
+    contentType,
+    longDescription: longDescription || null,
+    slug,
+    externalLink: externalLink || null,
+    level: level || null,
+    durationHours: durationHours === undefined ? null : durationHours,
```

Comment:
can we just do `durationHours: durationHours || null`

Reply:
yeah I was thinking since it's a number, we'd still want 0 to be represented, but I guess if it's 0 it would be a mistake and shouldn't be in there anyway.